Tasks/ArchiveFilesV2/Tests/L0.ts

```diff
@@ -41,7 +41,7 @@ describe('ArchiveFiles L0 Suite', function () {
     
     tl.setResourcePath(path.join( __dirname, '..', 'task.json'));
     cases.forEach(function(numberOfFiles) {
-        it('Verify plan output for ' + numberOfFiles + ' files has correct number of lines', (done: MochaDone) => {
+        it('Verify plan output for ' + numberOfFiles + ' files has correct number of lines', (done: Mocha.Done) => {
```

Comment:
Nit: Could you replace strings contention by template literal?

Reply:
Thanks! Updated